src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java

```diff
@@ -67,19 +71,34 @@ private void editDraftOrder(PageBuilder pageBuilder) {
             .done();
     }
 
+    private void previewOrder(PageBuilder pageBuilder) {
+
+        pageBuilder
+            .page("previewOrder", this::aboutToSubmit)
+            .pageLabel("Preview order")
+            .label("previewDraft", " Order preview")
+            .label("make Changes", "To make changes, choose 'Edit order'\n\n"
+                + "If you are happy , continue to the next screen.")
+            .done();
+    }
+
+
+    public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
+        final CaseDetails<CaseData, State> details,
+        final CaseDetails<CaseData, State> beforeDetails
+    ) {
 
-    public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit() {
 
         return AboutToStartOrSubmitResponse.<CaseData, State>builder()
-            .state(State.NewCaseReceived)
+            .state(State.Draft)
             .build();
 
     }
 
     public SubmittedCallbackResponse draftCreated(CaseDetails<CaseData, State> details,
                                                   CaseDetails<CaseData, State> beforeDetails) {
         return SubmittedCallbackResponse.builder()
-            .confirmationHeader("Draft order edited")
+            .confirmationHeader(format("When you have finished drafting this order,you can send it to parties in this case."))
             .build();
```

Comment:
Doesn't String.format expect some specifiers in the String? and if they aren't there (It's just a literal), you don't need to use format?